lib/manager/sbt/readme.md

```diff
@@ -264,4 +264,6 @@ N/A.
 
 #### Is there anything else to know about this package manager?
 
-N/A.
+It doesn't support Scala version inference well (i.e. `%%` operator).
+The manager just interpolate `scalaVersion` value to resulting package name.
+In case of problems, please use explicit versions.
```

Comment:
Is it always possible to use an explicit version instead of an interpolated one? Also, I'm thinking now we should move or duplicate this documentation into the website doc for the `sbt` object.